### ISSUE TITLE
WCAG 1.3.1,WCAG 3.3.2: Ensures every form element has a label (#agencyCd)

### ISSUE THREAD

**Issue by oklimanova** [oklimanova/hello-world#6]
#### Issue
`Ensures every form element has a label` ([label](https://dequeuniversity.com/rules/axe/3.5/label?application=msftAI))
#### Target application
[Current Unified Agenda of Regulatory and Deregulatory Actions](http://localhost:8080/public/do/eAgendaMain)
#### Element path
#agencyCd
#### Snippet
`<select id="agencyCd" name="agencyCd">`
#### How to fix
    Fix any of the following:
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Form element does not have an implicit (wrapped) <label>
      Form element does not have an explicit <label>
      Element has no title attribute or the title attribute is empty
#### Environment
Chrome version 81.0.4044.43
This accessibility issue was found using Accessibility Insights for Web 2.15.0 (axe-core 3.5.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
